src/mpyl/steps/deploy/k8s/helm.py

```diff
@@ -60,28 +71,86 @@ def __remove_existing_chart(
     )
 
 
-def install(
+def __execute_install_cmd(
     logger: Logger,
-    chart: dict[str, CustomResourceDefinition],
     step_input: Input,
     chart_name: str,
     name_space: str,
     kube_context: str,
     delete_existing: bool = False,
+    additional_args: str = "",
 ) -> Output:
     if delete_existing:
         removed = __remove_existing_chart(logger, chart_name, name_space, kube_context)
         if not removed.success:
             return removed
 
+    cmd = f"helm upgrade -i {chart_name} -n {name_space} --kube-context {kube_context} {additional_args}"
+    if step_input.dry_run:
+        cmd = (
+            f"helm upgrade -i {chart_name} -n namespace --kube-context {kube_context} {additional_args} "
+            f"--debug --dry-run"
+        )
+
+    logger.debug(f"Executing: {cmd}")
```

Comment:
These logs actually seem redundant since `custom_check_output` already does `logger.info(f"Executing: '{command_argument}'")`

Reply:
you are right!